speech/tests/system/gapic/v1/test_system_speech_v1.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Google LLC
+# Copyright 2019 Google LLC
```

Comment:
Please change this back to 2018. We keep the year the file was originally written in.